GLOSSARY.md

```diff
@@ -0,0 +1,21 @@
+## Glossary
+
+[Early draft, please add new definitions, fix the existing ones, add links and correct organizational details!]
+
+* **AFDKO** [Adobe Font Development Kit for OpenType](http://adobe-type-tools.github.io/afdko/AFDKO-Overview.html).
+* **AHG** ad-hoc group, a group of people representing multiple stakeholders, who meet informally to discuss technical proposals (example: the AHG who got together in 2015–2016 and made OpenType Variations happen), perhaps bypassing corporate policies on formal meetings.
+* **CG** Community Group, in a font technology context probably the W3C Font and Text Community Group established 2020.
+* **FEA** OpenType Feature File (extension .fea), [afdko: OpenType™ Feature File Specification](http://adobe-type-tools.github.io/afdko/OpenTypeFeatureFileSpecification.html).
+* **FEE** Font Engineering language with Extensibility, a language for creating font feature code that can be compiled to FEA using [fontFeatures](https://github.com/simoncozens/fontFeatures/blob/master/docs/fee-format.rst).
+* **FTCG** the W3C Font and Text Community Group.
+* **IEC** International Electrotechnical Commission: alongside ISO, jointly responsible for the OFF specification.
+* **ISO** International Organization for Standardization, ultimately responsible for the OFF specification via the MPEG Working Group.
+* **MOFF** “Microsoft Open Font Format”, meaning the same as Open Font Format but emphasizes that the specification is currently controlled by Microsoft.
```

Comment:
Hmm, perhaps redundant, regarding "MSOT". I don't think people actually use "Microsoft open font format" (see my OFF comment) but "Microsoft Opentype" is quite often used.

Reply:
MOFF is just Behdad being snarky. ;-)

Reply:
The only place I've seen this used is by @davelab6 on the MPEG-OTSPEC list and various other places. I'm not sure it's necessary or even *quite* accurate.

> The AHG doesn't own MSOT, MS does; and the W3C CG doesn't own MOFF, ISO does.

I added the caveat "quite" above because given that the OFF is roughly just a mirror of MSOT with the names changed to reflect ISO ownership, pragmatically speaking it is more under M$ control than any other entity, but on a technicality ISO does own the copyright and official approval process, hence I don't really see the utility of saying "MOFF" rather than just "OFF" in the same way that there is clearly utility to "MSOT" as a reminder that "OT" is not really open. Perhaps "ISOOFF" would be better.

Given it has been used and causes some confusion including it in a glossary isn't completely unreasonable, but mabye if we can convince Dave that it isn't a good utilitarian abbreviation we could just drop it.

Reply:
Scratch that this entry is just wrong. It should be "MPEG Open Font Format"

Reply:
Damn, got the number wrong it is : ISO/IEC 14496-22 “Open Font Format” , not 14996-22